### PR TITLE
Render embeds into campaign update blocks.

### PR DESCRIPTION
**Pull request by DFurnes**
#### Changes
This pull request adds support for rendering the newly-added "Link" field on Campaign Update blocks. This will allow campaign leads to display videos or article links in updates.

Check it out on [the review app](https://phoenix-next-pr-56.herokuapp.com/campaigns/teens-for-jeans) for a preview! 👀

### CODE REVIEW

**Comment by deadlybutter** [comment]
@DFurnes I'm really disappointed you didn't embed an overwatch play of the game from [my extensive library ](https://www.youtube.com/user/ThedeadlyButter/videos)

File: resources/assets/components/Embed/index.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+  render() {
+    let embed = <div className="spinner"/>;
+
+    // If an <iframe> code snippet is provided, use that. Otherwise, build preview card.
+    if (this.state.code) {
+      embed = <div className="media-video" dangerouslySetInnerHTML={{__html: this.state.code }} />;
+    } else if (this.state.title && this.state.url && this.state.image) {
+      embed = (
+        <a href={this.state.url}>
```
Comment (deadlybutter) [inline resources/assets/components/Embed/index.js]: Can we add the `target="_blank"` here so it does the new tab?

File: resources/assets/components/Embed/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          <Figure className="embed__preview" image={this.state.image} alignment="left" size="large">
+            <h3>{ this.state.title }</h3>
+            <p>{ this.state.description }</p>
```
Comment (weerd) [inline resources/assets/components/Embed/index.js]: Could the `this.state.description` be unassigned? If so, maybe us a ternary?

File: resources/assets/components/Embed/index.js (hunk 1 above)
Comment (DFurnes) [inline resources/assets/components/Embed/index.js]: Good call!

File: resources/assets/components/Embed/index.js (hunk 2 above)
Comment (DFurnes) [inline resources/assets/components/Embed/index.js]: That's a good thought. Will do. 🛠

File: resources/assets/components/Embed/index.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+    // If an <iframe> code snippet is provided, use that. Otherwise, build preview card.
+    if (this.state.code) {
+      embed = <div className="media-video" dangerouslySetInnerHTML={{__html: this.state.code }} />;
+    } else if (this.state.title && this.state.url && this.state.image) {
+      embed = (
+        <a href={this.state.url} target="_blank">
+          <Figure className="embed__preview" image={this.state.image} alignment="left" size="large">
+            <h3>{ this.state.title }</h3>
+            <p>{ this.state.description }</p>
```
Comment (weerd) [inline resources/assets/components/Embed/index.js]: Could the this.state.description be unassigned? If so, maybe us a ternary?

**Pull request merged**